Dockerfile

```diff
@@ -0,0 +1,39 @@
+
+FROM node:14-alpine as dependencies
+WORKDIR /app
+COPY package.json .
+RUN npm install
+
+FROM node:14-alpine as build
+# These vars will be sourced during build time
```

Comment:
these are not the correct env variables.I have DM you the valid env variables

Reply:
Updating the PR asap